crates/types/src/config/bifrost.rs

```diff
@@ -56,6 +56,15 @@ pub struct BifrostOptions {
     #[serde_as(as = "serde_with::DisplayFromStr")]
     #[cfg_attr(feature = "schemars", schemars(with = "String"))]
     pub seal_retry_interval: humantime::Duration,
+
+    /// # Append Retry policy
+    ///
+    /// Retry policy to use when bifrost waits for reconfiguration to complete during
+    /// append operations.
+    ///
+    /// Note that appends will be retried forever and setting `max-attempts` have no impact
+    /// on this particular retry policy.
```

Comment:
Don't we stop retrying in `wait_next_unsealed_loglet` and fail with `Error::LogSealed(log_id)` if the retry policy is exceeded?

Reply:
Fixed.